scripts/python_compliance.sh

```diff
@@ -1,60 +1,60 @@
-echo "Running compilance check"
+echo "Running compliance check"
 
 PARENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null 2>&1 && pwd )"
 
 pushd $PARENT_DIR
 
-if ! command -v mamba &> /dev/null
+if ! command -v micromamba &> /dev/null
```

Comment:
In case someone is still using mamba and not micro mamba, I guess forcing them to upgrade isn't to harmful :)

Reply:
Then we'd need to update some READMEs as well. I'd actually prefer a brief logic that uses micromamba preferably, but if it does not exist checks for mamba to not break everyone's setup (again after the conda switch :D). We can of course emit a warning/hint to get micromamba.

Reply:
This is still open I think